contracts/cw20-merkle-airdrop/schema/config_response.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please remove msg.rs above (empty file), as you use src/msg.rs

Reply:
Oh left over code.